modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -586,7 +586,7 @@ public void testAddOrUpdateSystemObjectDefinition() throws Exception {
 
 					@Override
```

Comment:
@leilanyaragao could you run the SF in this commit, please? You removed some tabs by mistake.
Please, use this [PR](https://github.com/liferay-objects/liferay-portal/pull/2239) as example.